src/code/z_kankyo.c

```diff
@@ -698,12 +1272,388 @@ s32 Environment_IsSceneUpsideDown(PlayState* play) {
     return ret;
 }
 
+#define LERP16(x, y, scale) ((s16)(((y) - (x)) * (scale)) + (x))
```

Comment:
Should we move this with the other LERP macros?

Reply:
Sure. Also renamed to `S16_LERP` to be consistent with `F32_LERP`